com.unity.render-pipelines.high-definition/Editor/Wizard/HDProjectSettings.cs

```diff
@@ -34,9 +44,51 @@ enum Version
         bool m_WizardNeedRestartAfterChangingToDX12 = false;
         [SerializeField]
         bool m_WizardNeedToRunFixAllAgainAfterDomainReload = false;
+
+        // Note: this is for the *material version*, which uses MaterialPostProcessor k_Migrations[] as migration
+        // functions table. See note on Version enum above.
         [SerializeField]
         int m_LastMaterialVersion = k_NeverProcessedMaterialVersion;
 
+        [SerializeField]
+        ShaderGraphVersion m_HDShaderGraphLastSeenVersion = ShaderGraphVersion.NeverMigrated;
+
```

Comment:
The above is for the MaterialPostProcessor to rescan to project for shadergraph changes in the same way that we do for material. This is currently disabled though, see s_AutoReimportProjectShaderGraphsOnVersionUpdate in the MaterialPostProcessor.